circle.yml

```diff
@@ -0,0 +1,38 @@
+machine:
+  environment:
+    ANDROID_HOME: /usr/local/android-sdk-linux
+    ANDROID_BUILD_TOOLS: 24.0.1
+    ANDROID_PLATFORM: 24
+
+dependencies:
+  pre:
+    - openssl aes-256-cbc -d -in ci-keys.cipher -k $CI_SECRETS_KEY >> ~/.circlerc
+    - openssl aes-256-cbc -d -in release.keystore.cipher -out release.keystore -k $CI_SECRETS_KEY
+    - if [ ! -e $ANDROID_HOME/build-tools/$ANDROID_BUILD_TOOLS ]; then echo y | android update sdk --no-ui --all --filter build-tools-$ANDROID_BUILD_TOOLS; fi
+    - if [ ! -e $ANDROID_HOME/platforms/android-$ANDROID_PLATFORM ]; then echo y | android update sdk --no-ui --all --filter "android-${ANDROID_PLATFORM}"; fi
+
+  cache_directories:
+    - /usr/local/android-sdk-linux/build-tools/24.0.1
+    - /usr/local/android-sdk-linux/platforms/android-24
+    - /usr/local/android-sdk-linux/tools
+
+test:
+  pre:
+    - if [ ! -n "${GCLOUD_SERVICE_KEY+1}" ]; then emulator -avd circleci-android24 -no-audio -no-window; fi:
+                background: true
+                parallel: true
+    - if [ ! -n "${GCLOUD_SERVICE_KEY+1}" ]; then circle-android wait-for-boot; fi
+
+  override:
+    - scripts/ci.sh $CIRCLE_TEST_REPORTS
```

Comment:
maybe a comment about what ci.sh is doing so you dont have to go look at both? 1st world problems 🌎